### PR TITLE
add method to move Trump to steak

### PR DESCRIPTION
**Pull request by viktoriaNadarajah**
Anders og Viktoria added a method to Trump to go to cleaning room and stays there for three rounds

### CODE REVIEW

File: src/maga/character/Trump.java [hunk 1]
```diff
@@ -14,25 +15,60 @@
  */
 public class Trump extends Character {
     /**
+     * Integer that sets how many rounds Trump should wait.
+     */
+    private int roundsToWait;
```
Comment (quartercastle) [inline src/maga/character/Trump.java]: This should be initialized with 0

File: src/maga/character/Trump.java [hunk 2]
```diff
@@ -14,25 +15,60 @@
...
+            System.out.println("You have to be in the cleaning room to use this method.");
+            return; 
+        }
+        if (player.hasItem("steak-with-ketchup")) {
+            this.setCurrentRoom(cleaningRoom);
+            roundsToWait = 3; 
+            player.dropItems(new Command(CommandWord.DROP, "steak-with-ketchup"));
+            player.goRoom(new Command(CommandWord.GO, "south"));
+            System.out.println("You called Trump to the cleaning room.");
+        }
```
Comment (quartercastle) [inline src/maga/character/Trump.java]: Would be nice to have a message if you don't have a steak with ketchup.
```java
if (!player.hasItem("steak-with-ketchup")) {
     System.out.println("You can't call trump without a steak with ketchup");
     return;
}

**Review by quartercastle — CHANGES_REQUESTED** [review]

File: src/maga/character/Trump.java (hunk 1 above)
Comment (viktoriaNadarajah) [reply src/maga/character/Trump.java]: fixed in: 871f79b749c3891c94e586cfcfecd9376b71ed2c

File: src/maga/character/Trump.java (hunk 2 above)
Comment (viktoriaNadarajah) [reply src/maga/character/Trump.java]: fixed in: 871f79b749c3891c94e586cfcfecd9376b71ed2c

**Review by quartercastle — APPROVED** [review]

**Pull request merged**